wix/main.wxs

```diff
@@ -62,16 +59,18 @@
                 <Directory Id='INSTALLDIR' Name='Volta'>
                 </Directory>
             </Directory>
+            <?if $(sys.BUILDARCH) = x64 ?>
             <Merge Id='VCRedist' SourceFile='wix\Microsoft_VC140_CRT_x64.msm' DiskId='1' Language='0'/>
+            <?endif ?>
         </Directory>
 
         <ComponentGroup Id='Binaries' Directory='INSTALLDIR'>
-            <Component Id='voltaBinary' Guid='*' Win64='$(var.Win64)'>
+            <Component Id='voltaBinary' Guid='*'>
```

Comment:
Here and throughout, why are you removing `Win64=`?

Reply:
I am deleting it for the same reason as here: https://github.com/volta-cli/volta/pull/1801#discussion_r1687443667